### ISSUE TITLE
"Error: RangeError: Too many properties to enumerate" producing large photo

### ISSUE THREAD

**Issue by Zony-Zhao** [naptha/tesseract.js#480]
Code is simple.
Generally, I get the image width and height, and divide into two rectangles according to the size. Then use the example page_segment_mode code to deal with it. The image is about 6k*4k, 20 MB, jpeg.
```
import {createScheduler, createWorker, PSM} from 'tesseract.js';
import path from "path";
import {writeFileSync} from "fs";
console.log( __dirname );
const imagePath='013.jpg'
const sizeOf = require('image-size');
const dimensions = sizeOf('013.jpg');
const pageWidth=dimensions.width/2
const pageHeight=dimensions.height/2
const rectangles=[
    {
        left: 0,
        top: 0,
        width: pageWidth,
        height: pageHeight,
    },
    {
        left: pageWidth,
        top: 0,
        width: pageWidth,
        height: pageHeight,
    }
]
console.log(dimensions.width, dimensions.height);
const scheduler = createScheduler();

(async () => {
    await Promise.all(new Array(2).fill(1).map(async (_,i)=>{
        const worker = createWorker( {
            langPath: path.join(__dirname, '..', 'langs'),
            logger: m => console.log(m),

        });
        await worker.load();
        await worker.loadLanguage("eng");
        await worker.initialize('eng');
        await worker.setParameters({
            tessedit_pageseg_mode: PSM.SINGLE_BLOCK,
        });
        scheduler.addWorker(worker);
        return worker
    }))
    const results = await Promise.all(rectangles.map((rectangle) => scheduler.addJob('recognize', imagePath, { rectangle })
    ));
    // writeFileSync("./result.txt",JSON.stringify(results))
    await scheduler.terminate();
})();

```
Error is weird. 
```
K:\project\nodejs\backNews\node_modules\tesseract.js\src\createWorker.js:173
        throw Error(data);
              ^
Error: RangeError: Too many properties to enumerate
    at ChildProcess.<anonymous> (K:\project\nodejs\backNews\node_modules\tesseract.js\src\createWorker.js:173:15)
    at ChildProcess.emit (events.js:315:20)
    at emit (internal/child_process.js:876:12)
    at processTicksAndRejections (internal/process/task_queues.js:85:21)

Process finished with exit code 1
```
I'm OK with the example image in doc. Maybe this photo too big? Is there any workaround?

**Comment by Balearica** [comment]
We implemented several changes in version 3.0.0 that significantly improve performance with large images.  If this issue is still active, please try again with the current version.

**Comment by Balearica** [comment]
Closing as stale and because I believe this was resolved.  Feel free to reopen if you encounter again.
